index.html

```diff
@@ -26,91 +26,219 @@ <h1>
+      answers: {
+        a: "9 lat",
+        b: "6 lat",
+        c: "3 lata",
+      },
+      correctAnswer: "a",
+    },
+    {
+      question:
+        "11. W roku 1996 tygrysy, pandy wielkie oraz czarne nosorożce znajdowały się na liście zwierząt zagrożonych. Które spośród tych gatunków obecnie sklasyfikowano jako bardziej zagrożone wyginięciem?",
+      answers: {
+        a: "wszystkie trzy",
+        b: "jeden",
+        c: "żaden",
+      },
+      correctAnswer: "c",
+    },
+    {
+      question:
+        "12. Jaki procent ludzi na świecie ma w jakimś stopniu dostęp do elektryczności",
+      answers: {
+        a: "20%",
+        b: "50%",
+        c: "80%",
+      },
+      correctAnswer: "c",
+    },
+    {
+      question:
+        "13. Klimatolodzy uważają, że przez następne 100 lat średnia temperatura na Ziemi:",
+      answers: {
+        a: "wzrośnie",
+        b: "pozostanie bez zmian",
+        c: "spadnie",
+      },
+      correctAnswer: "a",
+    },
+  ];
+  
 
-function QuestionItem({question, questionNumber, answers}){
-  return(
-    <fieldset> 
-      <legend>
-        {question}
-      </legend>
-      <div className='answers'>
-        {Object.entries(answers).map(([k, v], i) => 
-          <div className='answer' key={i}><label><input type='radio' name={questionNumber} value={k}/>{v}</label></div>)}
+function QuestionItem({ item, questionNumber }) {
+  return (
+    <fieldset>
+      <legend>{item.question}</legend>
+      <div className="answers">
+        {Object.entries(item.answers).map(([k, v], i) => (
+          <div className="answer" key={i}>
+            <label>
+              <input
+                type="radio"
+                name={"question_" + questionNumber}
+                value={k}
+              />
+              {v}
+            </label>
+          </div>
+        ))}
       </div>
     </fieldset>
   );
 }
 
-function Quiz(props){
-    const content = props.data;
-    const listItems = content.map((object,i) =>
-        <li key={i}>
-            <QuestionItem
-                question={object.question}
-                questionNumber={'question_'+i}
-                answers={object.answers}
-            />
-        </li>
-    );
-  return (
-    <ul>{listItems}</ul>
-  );
+function Quiz(props) {
+  const content = props.data;
+  const listItems = content.map((item, i) => (
+    <li key={i}>
+      <QuestionItem 
+        item={item}
+        questionNumber={i} 
+      />
+    </li>
+  ));
+  return <ul>{listItems}</ul>;
 }
 
 ReactDOM.render(
-  <Quiz data={data}/>,
-  document.getElementById('quiz')
+  <Quiz data={data} />,
+  document.getElementById("quiz")
 );
 
+addImg();
+
+function addImg(){
+  const items = document.getElementsByName('question_7');
+  const answers = items[0].parentElement.parentElement.parentElement;
+  const img = document.createElement('img');
+  img.src = 'img/mapy.png';
+  answers.prepend(img);
+}

```

Comment:
To proponuję zrobić, jak mówiłam, src przy odpowiednim `item` na liście i przy renderowaniu sprawdzasz, czy jest `item.src`, jak jest, to wyświetlasz w odpowiednim miejscu w divie.